governance/charter-and-members.md

```diff
@@ -0,0 +1,125 @@
+
+### Alex Jones
+
+<img src="https://i.ibb.co/M5SLpD0/KC-CNC-NA-headshot-221027-Alex-Jones-2215.jpg" height="200" />
+
+- Company: Canonical
+- GitHub: https://github.com/AlexsJones
+- Twitter: https://twitter.com/AlexJonesax
+- Personal Website: https://www.cloudnativeskunkworks.io/
+
+#### Description of candidate ####
+
+Alex works as both a contributor and end-user of cloud-native technology. 
+When not working as Kubernetes Engineering Director at Canonical; he contributes to CNCF TAG App Delivery as Tech Lead and to the Open Feature project on the governing board. He also maintains several other popular community projects.
+
+---
+
+### Alois Reitbauer
+
+<img src="https://engineering.dynatrace.com/_next/image?url=https%3A%2F%2Fbackend.engineering.dynatrace.com%2Fimg%2Fresized%2F291%3Fwidth%3D220&w=3840&q=90" alt="Alois Reitbauer" height="200" width="200">
+
+- Company: Dynatrace
+- GitHub: https://github.com/aloisreitbauer
+- Twitter: https://twitter.com/aloisreitbauer
+
+#### Description of candidate ####
+
+Alois works as Chief Technology Strategist at Dynatrace, where he leads all open source and research efforts. He is also co-chair of TAG App Delivery in the CNCF and contributor and maintainer to several other open source projects like OpenFeature and OpenTelemetry. Alois is the initial creator of the Keptn project and was driving most of the conceptual over the last couple of years. He is also a co-creator of the application lifecycle toolkit for Keptn. 
+
+---
+
+### Ana Margarita Medina
+
+<img src="https://avatars.githubusercontent.com/u/3894791" alt="Ana Margarita Medina" height="200" width="200">
+
+- Company: Lightstep
+- GitHub: [AnaMMedina21](https://github.com/AnaMMedina21)
+- Twitter [@Ana_M_Medina](https://twitter.com/Ana_M_Medina)
+- LinkedIn [Ana Margarita Medina](https://www.linkedin.com/in/anammedina/)
+
+#### Description of candidate ####
+Ana Margarita is a Staff Developer Advocate at Lightstep and focuses on helping companies be more reliable by leveraging Observability and Incident Response practices. She is also on the Kubernetes Release Team (v1.25 - v1.26) and has been advising CNCF's Keptn project since 2019. Before Lightstep, she was a Senior Chaos Engineer at Gremlin and helped companies avoid outages by running proactive chaos engineering experiments. She has also worked at various-sized companies including Google, Uber, SFEFCU, and Miami-based startups. Ana is an internationally recognized speaker and has presented at: AWS re:Invent, KubeCon, DockerCon, DevOpDays, AllDayDevOps, Write/Speak/Code, and many others. Catch her tweeting at @Ana_M_Medina about traveling, diversity in tech, and mental health.
+
+> NOTE:
+> In November 2022 we held our first elections.
+
+## Technical Committee (TC) Scope
+
+Role and responsibilities
+
+
+### TC Members
+
+
+
+### TC Charter
+
+
+
+## Decision making
+
+
+
+## Meetings
+
+
+
+### Project roles
+
+The following project roles are defined at the moment:
```

Comment:
These are a lot of roles ... Do we really need all of them?

Reply:
it would leave space for intermediate steps